pandas/tests/arithmetic/test_datetime64.py

```diff
@@ -1155,14 +1155,18 @@ def test_dti_add_intarray_no_freq(self, box):
     def test_dti_add_timedeltalike(self, tz_naive_fixture, two_hours,
                                    box_with_datetime):
         # GH#22005, GH#22163 check DataFrame doesn't raise TypeError
+        box = box_with_datetime
+
         tz = tz_naive_fixture
         rng = pd.date_range('2000-01-01', '2000-02-01', tz=tz)
-        rng = tm.box_expected(rng, box_with_datetime)
+
+        # FIXME: calling with transpose=True raises ValueError
+        rng = tm.box_expected(rng, box, transpose=False)
```

Comment:
Can we potentially have `xfail` tests for these?

Reply:
Good idea.  I've implemented this for some of these (will push shortly).  For others (particularly ones using box_with_period or box_with_datetimes instead of box) doing that becomes much hairier and I'm inclined to leave the FIXME notes for the time being.